package.json

```diff
@@ -20,6 +20,7 @@
     "@npmcli/eslint-config": "^5.0.1",
     "@npmcli/template-oss": "4.25.0",
     "mutate-fs": "^2.1.1",
+    "proc-log": "^5.0.0",
```

Comment:
As the linting error mentioned, this is a `dependency` since it is used by the package itself as installed.